### PR TITLE
table_grant: update table grants in-place instead of recreating

### PR DESCRIPTION
**Pull request by igungor**
Related to #469
Updates #137

### CODE REVIEW

File: pkg/resources/table_grant.go [hunk 1]
```diff
@@ -42,28 +42,24 @@ var tableGrantSchema = map[string]*schema.Schema{
 		Description:  "The privilege to grant on the current or future table.",
 		Default:      privilegeSelect.String(),
 		ValidateFunc: validation.ValidatePrivilege(validTablePrivileges.ToList(), true),
-		ForceNew:     true,
```
Comment (edulop91) [inline pkg/resources/table_grant.go:45]: I might be missing something but it seems to me we should keep `ForceNew = true` here. The update function doesn't currently know how to handle `privilege` or `on_future` updates.

**Review by edulop91 — COMMENTED** [review]
one small comment, otherwise lgtm. Thank you

File: pkg/resources/table_grant.go (hunk 1 above)
Comment (igungor) [reply pkg/resources/table_grant.go:45]: yes, we should keep this as is. reverting now.

**Review by edulop91 — APPROVED** [review]

**Pull request merged**